addons/godot-xr-tools/interactables/interactable_slider.gd

```diff
@@ -96,13 +96,13 @@ func move_slider(var position: float) -> void:
 
 
 # Handle release of slider
-func _on_slider_released(var _interactable):
+func _on_slider_released(_interactable):
```

Comment:
No type here?

Reply:
Added typing for the interactable being release (always self for this self-subscribing handler)